net/sscep/Makefile

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (C) 2006 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
```

Comment:
Remove it or assign it to yourself.
Check: https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#makefile-contents-should-contain